hw2/Views/LeagueView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  LeagueView.swift
+//  hw2
+//
+//  Created by Tina Jureško on 14.03.2025..
+//
+
+import Foundation
+import UIKit
+import SofaAcademic
+import SnapKit
+
+class LeagueView: BaseView {
+    
+    private let logoImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "LaLiga")
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+    
+    private let countryLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private let arrowImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "Vector")
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        return imageView
+    }()
+    
+    private let nameLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+        
+    override func addViews() {
+        super.addViews()
+        addSubview(logoImageView)
+        addSubview(countryLabel)
+        addSubview(arrowImageView)
+        addSubview(nameLabel)
+    }
+
+    override func styleViews() {
+        logoImageView.contentMode = .scaleAspectFit
+        arrowImageView.contentMode = .scaleAspectFit
+        
+        countryLabel.font = .regularBold14()
+        countryLabel.textColor = AppStyles.Colors.primary
+        countryLabel.numberOfLines = 0
+        countryLabel.lineBreakMode = .byWordWrapping
+        
+        nameLabel.font = .regularBold14()
+        nameLabel.textColor = AppStyles.Colors.secondary
+        nameLabel.numberOfLines = 0
+        nameLabel.lineBreakMode = .byWordWrapping
+    }
+        
+    override func setupConstraints() {
+        super.setupConstraints()
+        
+        logoImageView.snp.makeConstraints {
```

Comment:
Dobro je stavljati u ostalim viewovima $0.centerY.equalTo(logoImageView) i tako dalje, ali barem jedan view se mora zapeci za top i bottom. A to ti moze biti ovaj, ako mu je size 32, njega stavis top.bottom.equealToSuperview().inset(12)

I onda zanas da ce ti view biti height 56... 12 + 12 + 32.